environment-api/src/main/java/com/sequenceiq/environment/api/v1/environment/model/request/AttachedFreeIpaRequest.java

```diff
@@ -31,6 +32,10 @@ public class AttachedFreeIpaRequest implements Serializable {
     @ApiModelProperty(value = EnvironmentModelDescription.MULTIAZ_FREEIPA)
     private boolean enableMultiAz;
 
+    @Valid
```

Comment:
what does `@Valid` mean for a set of string?

Reply:
Nothing :) I just left it here